File: webapp/HIN_WEB/Models/Networking.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace HIN_WEB.Models
{
    public class Networking
    {
        public int NetworkingId { get; set; }
        public string NetworkingName { get; set; }
        public string NetworkingNumber { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Zipcode { get; set; }
        public string Website { get; set; }
        public string Telephone { get; set; }
        public string Fax { get; set; }
        public string Country { get; set; }
        public string Industry { get; set; }
        public int? CostId { get; set; }
        public int? EventMeetId { get; set; }
        public bool? Inactive { get; set; }
        public NetworkingCost Cost { get; set; }
        public NetworkingEventMeet EventMeet { get; set; }
        public DateTime? UpdatedOn { get; set; }
        public string UpdatedBy { get; set; }
        public int? UpdatedById { get; set; }
        public DateTime? CreatedOn { get; set; }
        public string CreatedBy { get; set; }
        public int? CreatedById { get; set; }
    }
}

File: webapp/HIN_WEB/ClientApp/src/app/model/networking.ts
import { Networkingcost } from "./networkingcost";
import { Time } from "@angular/common";

export class Networking {
  public NetworkingId: number;
  public NetworkingName: string;
  public NetworkingNumber: string;
  public Address: string;
  public City: string;
  public State: string;
  public Zipcode: string;
  public Website: string;
  public Telephone: string;
  public Fax: string;
  public Country: string;
  public Industry: string;
  public CostId: number;
  public EventMeetId: number;
  public Inactive: boolean;
  public Cost: Networkingcost;
  public EventMeet: NetworkingEventMeet;
  public UpdatedOn: Date;
  public UpdatedBy: string;
  public UpdatedById: number;
  public CreatedOn: Date;
  public CreatedBy: string;
  public CreatedById: number;
}
export class NetworkingEventMeet {
  public Id: number;
  public YearlyDate: Date;
  public MonthlyDate: Date;
  public WeeklyDate: Date;
  public YearlyTime: Time;
  public MonthlyTime: Time;
  public WeeklyTime: Time;
  public Notes: string;
  public UpdatedOn: Date;
  public UpdatedBy: string;
  public UpdatedById: number;
  public CreatedOn: Date;
  public CreatedBy: string;
  public CreatedById: number;
}
